### PR TITLE
Feature/chat image support

### PR DESCRIPTION
**Pull request by wjutsum**
This PR adds support for images in chat messages.

Example command:

`claudsync chat message --chat {chat-id} --image test.jpg "{chat-message}"`

### CODE REVIEW

File: src/claudesync/providers/claude_ai.py [hunk 1]
```diff
@@ -136,3 +146,79 @@ def _make_request_stream(self, method, endpoint, data=None):
...
+        headers = {
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0",
+            'Accept': '*/*',
+            'Accept-Language': 'en-GB,en;q=0.9',
+            'anthropic-client-sha': 'unknown',
+            'anthropic-client-version': 'unknown',
+            'origin': 'https://claude.ai',
+            'referer': 'https://claude.ai/',
+            'Content-Type': f'multipart/form-data; boundary={boundary}',
+            'Cookie': f'sessionKey={self.config.get_session_key("claude.ai")[0]}'
```
Comment (jahwag) [inline src/claudesync/providers/claude_ai.py:187]: would you mind assigning this variable?

File: src/claudesync/providers/claude_ai.py [hunk 2]
```diff
@@ -136,3 +146,79 @@ def _make_request_stream(self, method, endpoint, data=None):
...
+
+        boundary = self._generate_boundary()
+        data = self._encode_multipart_formdata(
+            file_data, file_name, content_type, boundary)
+
+        headers = {
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0",
+            'Accept': '*/*',
+            'Accept-Language': 'en-GB,en;q=0.9',
+            'anthropic-client-sha': 'unknown',
```
Comment (jahwag) [inline src/claudesync/providers/claude_ai.py:182]: are these two anthropic headers needed?

**Review by jahwag — CHANGES_REQUESTED** [review]
Looks good, works well!

Some small details improvements/questions for you.

Please bump version in pyproject.toml to prep release 0.6.2.

File: src/claudesync/providers/claude_ai.py (hunk 1 above)
Comment (wjutsum) [reply src/claudesync/providers/claude_ai.py]: Done

File: src/claudesync/providers/claude_ai.py (hunk 2 above)
Comment (wjutsum) [reply src/claudesync/providers/claude_ai.py]: Done

**Comment by wjutsum** [comment]
Commenting to remove stale flag

**Review by jahwag — CHANGES_REQUESTED** [review]
this branch is behind master (0.6.2) and needs to be rebased and version bumped

**Comment by wjutsum** [comment]
> this branch is behind master (0.6.2) and needs to be rebased and version bumped

That's done @jahwag

**Pull request closed without merging**